ui/tract-renderer/src/main/java/org/cru/godtools/tract/activity/TractActivity.java

```diff
@@ -163,12 +164,14 @@ public static Intent createIntent(@NonNull final Context context, @NonNull final
                                       @NonNull final Locale... languages) {
         final Bundle extras = new Bundle();
         populateExtras(extras, toolCode, languages);
-        return new Intent(context, TractActivity.class).putExtras(extras);
+        return new Intent(context, TractActivity.class).putExtras(extras); //.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
     }
 
-    public static void start(@NonNull final Context context, @NonNull final String toolCode,
+    public static void start(@NonNull final Activity activity, @NonNull final String toolCode,
                              @NonNull final Locale... languages) {
-        context.startActivity(createIntent(context, toolCode, languages));
+        final Bundle extras = new Bundle();
+        populateExtras(extras, toolCode, languages);
+        activity.startActivity(new Intent(activity, TractActivity.class).putExtras(extras));
```

Comment:
you should still utilize the `createIntent()` method so we aren't duplicating this logic